tests/validate_tests.rs

```diff
@@ -0,0 +1,512 @@
+pub mod common;
+
+use aa_bundler::types::user_operation::UserOperation;
+use common::gen::{
+    EntryPointContract, TestOpcodesAccount, TestOpcodesAccountFactory, TestRulesAccount,
+    TestRulesAccountFactory, TestStorageAccountFactory,
+};
+use common::DeployedContract;
+use ethers::abi::Token;
+use ethers::prelude::BaseContract;
+use ethers::providers::{Http, Middleware};
+use ethers::types::transaction::eip2718::TypedTransaction;
+use ethers::types::Address;
+use ethers::utils::{parse_units, AnvilInstance};
+use ethers::{
+    core::utils::Anvil,
+    prelude::SignerMiddleware,
+    providers::Provider,
+    signers::{LocalWallet, Signer},
+    types::{Bytes, U256},
+};
+use std::ops::Deref;
```

Comment:
unused import?

Reply:
It is used in https://github.com/Vid201/aa-bundler/blob/4d55d1d3ea52d2780ad0cdd0c1d79af3758c1ccc/tests/validate_tests.rs#L112.
Currently, clippy should be enough for unused import.